packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraTestUtils.m

```diff
@@ -29,6 +29,51 @@
                                       error:nil];
 }
 
+FLTCam *FLTCreateCamWithVideoCaptureSession(AVCaptureSession *captureSession,
+                                            NSString *resolutionPreset) {
+  id inputMock = OCMClassMock([AVCaptureDeviceInput class]);
+  OCMStub([inputMock deviceInputWithDevice:[OCMArg any] error:[OCMArg setTo:nil]])
+      .andReturn(inputMock);
+
+  id audioSessionMock = OCMClassMock([AVCaptureSession class]);
+  OCMStub([audioSessionMock addInputWithNoConnections:[OCMArg any]]);  // no-op
```

Comment:
Same question here and below. (If this is meant to say that this is stubbing it out with a no-op, the comment should be more explicit.)

Reply:
removed